Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -386,13 +401,58 @@
 
 		% For Time-encoded, we skip the first volume per block
 		if x.modules.asl.bTimeEncoded
-			[PLDs, index] = unique(PLDs, 'stable');
-			LabDurs = LabDurs(index)';
+			% In case we are merging sessions, we have to read all JSONs again from files
+			if bMergingSessions
```

Comment:
Note that session merging is not only for time encoded. Either you change that here or in #1543 (in which case it should be added to that list)